router/transformer/transformer.go

```diff
@@ -150,11 +157,106 @@ func (trans *HandleT) Transform(transformType string, transformMessage *types.Tr
 	return destinationJobs
 }
 
+func (trans *HandleT) ResponseTransform(ctx context.Context, responseData integrations.DeliveryResponseT, destName string) (statusCode int, respBody string) {
+	rawJSON, err := json.Marshal(responseData)
+	requestFailed := false
+	retryCount := 0
+	if err != nil {
+		panic(err)
+	}
+	var resp *http.Response
+	var respData []byte
+	var respCode int
+	var tempRespData []byte
+	url := getResponseTransformURL(destName)
+	for {
+		trans.transformerResponseTransformRequestTime.Start()
+		resp, err = trans.client.Post(url, "application/json; charset=utf-8", bytes.NewBuffer(rawJSON))
+		trans.transformerResponseTransformRequestTime.End()
```

Comment:
Using timer start/end is not thread safe. We will get incorrect reading if it runs concurrently.

```suggestion
		s := time.Now()
		resp, err = trans.client.Post(url, "application/json; charset=utf-8", bytes.NewBuffer(rawJSON))
		
		trans.transformerResponseTransformRequestTime.SendTiming(time.Since(s))
```

Reply:
This is addressed